railties/lib/rails/application/configuration.rb

```diff
@@ -21,7 +21,7 @@ class Configuration < ::Rails::Engine::Configuration
                     :read_encrypted_secrets, :log_level, :content_security_policy_report_only,
                     :content_security_policy_nonce_generator, :content_security_policy_nonce_directives,
                     :require_master_key, :credentials, :disable_sandbox, :add_autoload_paths_to_load_path,
-                    :rake_eager_load, :server_timing
+                    :rake_eager_load, :server_timing, :log_size
```

Comment:
Maybe `log_file_size` instead of `log_size`? Otherwise it's a bit ambiguous I think.